src/app/features/secret-key-displayer/components/mnemonic-key/mnemonic-form.tsx

```diff
@@ -0,0 +1,127 @@
+import { OnboardingSelectors } from '@tests/selectors/onboarding.selectors';
+import { Form, Formik } from 'formik';
+import { css } from 'leather-styles/css';
+import { Flex, styled } from 'leather-styles/jsx';
+
+import { isEmpty } from '@shared/utils';
+
+import { createNullArrayOfLength } from '@app/common/utils';
+import { LeatherButton } from '@app/components/button/button';
+import { ErrorLabel } from '@app/components/error-label';
+import { SecretKeyGrid } from '@app/features/secret-key-displayer/components/secret-key-grid';
+import { useSignIn } from '@app/pages/onboarding/sign-in/hooks/use-sign-in';
+
+import { MnemonicWordInput } from './mnemonic-word-input';
+import {
+  getMnemonicErrorFields,
+  getMnemonicErrorMessage,
+  hasMnemonicFormValues,
+} from './utils/error-handling';
+import { validationSchema } from './utils/validation';
+
+export function MnemonicForm({
+  mnemonic,
+  setMnemonic,
+  twentyFourWordMode,
+}: {
+  mnemonic: Array<string | null>;
```

Comment:
I was pretty sure we had a lint error for enforcing `T[]` notation only, please stick to that for consistency

Reply:
Sorry the `T` was just an example, this fn has no generics. In this instance, the same type is `(string | null)[]`.

Reply:
Thanks Kyran! I struggle a bit with that stuff sometimes. 

I can at least admit it now that not liking TS became cool